qiskit/__init__.py

```diff
@@ -50,11 +51,16 @@
 try:
     from qiskit.providers.aer import Aer
 except ImportError:
+    warnings.warn('Could not import the Aer provider from the qiskit-aer '
+                  'package. Install qiskit-aer or check your installation.')
     pass
 # Try to import the IBMQ provider if installed.
 try:
     from qiskit.providers.ibmq import IBMQ
 except ImportError:
+    warnings.warn('Could not import the IBMQ account from the '
```

Comment:
`IBMQ provider` instead of `IBMQ account`? Also, the `pass` statements are no longer needed.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/2842/commits/baa365f2b2f3fead8093f7f889563bb6ca7a7a1b